environment.yml

```
@@ -3,27 +3,26 @@ channels:
     - conda-forge
     - defaults
 dependencies:
-    - python >=3.8
+    - python >=3.9,<3.12
```

```suggestion
    - python >=3.9
```
Is there a reason to pin python here and not simply let the package manager infer the pin themselves ?

---

It takes less time to resolve dependencies, that's all.